lib/ansible/module_utils/common/parameters.py

```diff
@@ -195,3 +195,29 @@ def handle_aliases(argument_spec, params, alias_warnings=None):
                 params[k] = params[alias]
 
     return aliases_results, legal_inputs
+
+
+def get_unsupported_parameters(argument_spec, module_parameters, legal_inputs=None):
+    """Check arguments in argument_spec against those provided in module_paramaters
```

Comment:
s/module_paramater/module_parameters/

Reply:
One day I will be able to spell that right consistently.